azavea/franklin
# Add HTML Landing Page

## Overview

This adds an HTML view for the landing page in the application. It pulls the most relevant parts from, adds some development instructions, and updates STRTA as well.

### Checklist


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

### Testing Instructions

- Run `./scripts/update` and then start the server and go to http://localhost:9090 and hopefully you see some HTML
- Review `development.md` and make sure the instructions/descriptions make sense

Closes #292
